test/e2e/framework/pod/utils_test.go

```diff
@@ -93,3 +94,57 @@ func TestMixinRestrictedPodSecurity(t *testing.T) {
 	})
 
 }
+
+func TestGetCPULimitCgroupExpectations(t *testing.T) {
```

Comment:
I'm not sure if we ever run these on windows, but this will fail if we do. Probably best to add 
```go
if runtime.GOOS == "windows" {
	t.Skip()
}
```

Reply:
done

Reply:
If this tests a Linux-only implementation of some method, shouldn't the entire test better be in a file which only gets built on Linux?

Reply:
yeah I think this relies on the [linux-only implementation of MilliCPUToQuota](https://github.com/kubernetes/kubernetes/blob/047e4c8e56b5c6a0466e4f1f1fdeca7e9a8de3d6/pkg/kubelet/cm/helpers_linux.go#L61)

Moved this test into utils_linux_test.go

Reply:
ah sorry I had forgotten to add the license header to the new file... just added it